python/pyspark/ml/classification.py

```diff
@@ -287,7 +287,7 @@ class DecisionTreeClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPred
     >>> stringIndexer = StringIndexer(inputCol="label", outputCol="indexed")
     >>> si_model = stringIndexer.fit(df)
     >>> td = si_model.transform(df)
-    >>> dt = DecisionTreeClassifier(maxDepth=2, labelCol="indexed")
+    >>> dt = DecisionTreeClassifier(maxDepth=2, labelCol="indexed", seed=42)
```

Comment:
Let's not specify seed in examples anymore (to be simpler) since it's now deterministic for spark.ml.  (If Python seed is None, then it will use the Scala HasSeed default, which is based on the class hashcode.)